### PR TITLE
verification: trusted channel -> second factor

### PR DESCRIPTION
**Pull request by azul**

### CODE REVIEW

File: source/new.rst [hunk 1]
```diff
@@ -124,26 +145,39 @@ messages.
...
 we assume that
-our active attacker *cannot* observe or modify data transferred via the
-trusted channel.
+our active attacker *cannot* observe or modify data transferred
+as the second factor.
+
+An attacker who can alter messages
+but has no way of reading or manipulating the second factor
+can disrupt the verification flow.
+
```
[the remark this answers is not in the document]
Comment (azul) [reply source/new.rst]: no, i actually mean it. 
An attacker who can alter the first channel can disrupt the verification flow.
If messages are dropped or altered verification will fail.

[the remark this answers is not in the document]
Comment (hpk42) [reply source/new.rst]: ah k.

File: source/new.rst [hunk 2]
```diff
@@ -124,26 +146,37 @@ messages.
...
 Recall that
 we assume that
-our active attacker *cannot* observe or modify data transferred via the
-trusted channel.
+our active attacker *cannot* observe or modify data transferred
+via the second channel.
+
+An attacker who can alter messages
+but has no way of reading or manipulating the second channel
+can disrupt the verification flow.
```
Comment (hpk42) [inline source/new.rst]: suggest to be more explicit: "Am attack who can alter messages on the first channel but has no way of reading or manipulating the second channel can prevent the verification protocol from completing successfully."

File: source/new.rst [hunk 3]
```diff
@@ -124,26 +146,39 @@ messages.
...
-our active attacker *cannot* observe or modify data transferred via the
-trusted channel.
+our active attacker *cannot* observe or modify data transferred
+via the second channel.
+
+An attacker who can alter messages
+but has no way of reading or manipulating the second channel
+can prevent the verification protocol
+from completing successfully
+by droping or altering messages.
```
Comment (hpk42) [inline source/new.rst]: "dropping"

Comment (azul) [reply source/new.rst]: fixed in master.

**Pull request merged**